pandas/core/arrays/base.py

```diff
@@ -596,7 +596,7 @@ def argsort(
             mask=np.asarray(self.isna()),
         )
 
-    def argmin(self):
+    def argmin(self, axis=None, skipna: bool = True):
```

Comment:
The `axis` keyword shouldn't be added, since it's being ignored. It can be accepted through args/kwargs, and then validated with our numpy compat (+ tests for the numpy compat)